frontend/javascripts/messages.tsx

```diff
@@ -73,6 +74,8 @@ export const settingsTooltips: Partial<Record<keyof RecommendedConfiguration, st
   mouseRotateValue: "Rotation speed when using the mouse to drag the rotation.",
   zoom: "Zoom in or out in the data viewports",
   displayScalebars: "Show a scale in the lower-right corner of each viewport",
+  blendMode:
+    "Set the blend mode for the dataset. The additive mode (default) adds the data values of all layers. The cover mode only shows the data of the top most layer in the list.",
```

Comment:
```suggestion
    "Set the blend mode for the dataset. The additive mode (default) adds the data values of all layers. In cover mode, layers are rendered on top of each other so that the data values of lower layers are hidden by values of higher layers.",
```

also: all layers or only color layers?